vtex/billing-app
# Change install behavior for free apps

#### What is the purpose of this pull request?
Change install behavior for free apps

#### What problem is this solving?
Installation of free apps through toolbelt has not been possible because of recent behavior of this app for declared free apps. This PR changes this by postponing the `isFree` verification so that it occurs after the user accepts the given app terms 

#### How should this be manually tested?
1. Clone/Pull this repo/branch and link this app in your workspace
2. Try to install a free app, such as `vtex.payment-provider-test-suite`
```
vtex install vtex.payment-provider-test-suite -r vtex
```
3. The toolbelt shows a message rendering the free billing options. Accept the terms
4. Your app should be successfully installed

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/3827456/41928689-6666d328-794c-11e8-895c-ba17d01b1aae.png)

#### Types of changes
